packages/react-components/src/stories/TextArea.stories.tsx

```diff
@@ -0,0 +1,98 @@
+import type { Meta, StoryObj } from "@storybook/react";
+
+import { TextArea } from "../components";
+
+const meta = {
+  title: "Components/TextArea",
+  component: TextArea,
+  parameters: {
+    layout: "centered",
+  },
+  argTypes: {
```

Comment:
Let's add `errorMessage` in here like this:

```js
errorMessage: {
  control: { type: "text" },
  description: "Error message text that appears when `isInvalid` is true", // Or whatever you want
},
```

That will still give us the `union` type tag on the Storybook page, but we can have access to that field in the Controls block:

<img width="1840" alt="TextArea Storybook Controls block showing errorMessage field" src="https://github.com/user-attachments/assets/4e5bb62f-7667-4ba8-a308-8ba5ea19eb51">

Reply:
Done in [40bfbdb](https://github.com/bcgov/design-system/pull/400/commits/40bfbdbcdfde0b1879b3a3373e9f147c8f142595)